src/notification/device.controller.ts

```
@@ -147,21 +156,33 @@ export class DeviceController {
   @UseGuards(AuthSessionGuard)
   @HttpCode(HttpStatus.OK)
   @ApiBearerAuth()
-  @ApiOperation({ summary: 'Unregister all devices (logout from all devices)' })
+  @ApiOperation({
+    summary: 'Unregister devices',
+    description:
+      'With a { token } body, unregisters only that device (the v1.2.0 logout ' +
+      'contract). With no body, unregisters all of the user\'s devices.',
```

ESLint

[error] Replace `'contract).·With·no·body,·unregisters·all·of·the·user\'s·devices.'` with `"contract).·With·no·body,·unregisters·all·of·the·user's·devices."`

(prettier/prettier)